### PR TITLE
Document update. Commit on behalf of Don(v-donglo2@microsoft.com).

### PR DESCRIPTION
**Pull request by NonStatic2014**
Update swagger with refined wording to prepare for new document web site.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/719)

### CODE REVIEW

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 1]
```diff
@@ -24,27 +24,39 @@
...
             "schema": {
               "$ref": "#/definitions/WebService"
             }
-          }
+          }                
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. This respoonse is returned for an update web service operation. The response payload is identical to the response payload that is returned by the GET operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more informationFor more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Typo in "respoonse"

Comment (NonStatic2014) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Good catch!

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 2]
```diff
@@ -24,27 +24,39 @@
...
             "schema": {
               "$ref": "#/definitions/WebService"
             }
           },
           "201": {
-            "description": "Created",
+            "description": "Created. This response is returned for a create web service operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more informationFor more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: "For more information" is duplicated

Comment (NonStatic2014) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: They are different response code. That's why we write twice.

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 3]
```diff
@@ -203,7 +232,7 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. The response includes a paginated array of web service objects and a URI to the next set of results, if any. For the more infomartion the limits of the number of items in a resource group, see https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/. Note that the web service objects are sparsely populated to conserve space in the response content. To get the full web service object, call the GET operation on the web service.",
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Typo in "infomartion"

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 4]
```diff
@@ -358,56 +400,56 @@
         },
         "keys": {
           "$ref": "#/definitions/WebServiceKeys",
-          "description":  "The set of access keys for the web service. If not specified at creation time (PUT), they will be generated automatically by the resource provider."
+          "description":  "Contains the web service provisioning keys. If you do not specify provisioning keys, the Azure Machine Learning system generates them for you. Note: The keys are not returned from calls to GET operations."
         },
         "readOnly": {
           "type": "boolean",
-          "description": "If true, the web service can no longer be updated / patched, only removed. Otherwise, the service resource supports changes."
+          "description": "When set to true, indicates that the Web Service is read-only and can no longer be updated or patched, only removed. Default, is false. Note: Once set to true, you cannot change its value."
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Do you want "web service", for consistency?

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 5]
```diff
@@ -358,56 +400,56 @@
...
+          "description": "Contains the configuration settings for the web service endpoint."
         },
         "diagnostics": {
           "$ref": "#/definitions/DiagnosticsConfiguration",
           "description": "Settings controlling the diagnostics traces collection for the web service."
         },
         "storageAccount": {
           "$ref": "#/definitions/StorageAccount",
-          "description": "The storage account associated with the service. This is used to store both datasets and diagnostic traces. This information is required at creation time (PUT) and only the key is updateable after that. The account credentials are hidden on a GET web service call."
+          "description": "Specifies the storage account that Azure Machine Learning uses to store information about the web service. Only the name of the storage account is returned from calls to GET operations. When updating the storage account information, you must ensure that all necessary assests are available in the new storage account or calls to your web service will fail."
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Typo in "assests"

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json (hunk 4 above)
Comment (NonStatic2014) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Will change it to lower case.

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 6]
```diff
@@ -546,7 +588,7 @@
       "properties": {
         "id": {
           "type": "string",
-          "description": "The commitment plan ARM resource  id."
+          "description": "Specifies the Azure Rerource Manager ID of the commitment plan associated with the web service."
```
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Typo in "Rerource"

**Review by BryanLa — COMMENTED** [review]
Completed review of descriptions, but SDK team will need to review schema changes.

**Review by BryanLa — APPROVED** [review]
@amarzavery - the description changes are done; can you sign off on the other changes and merge?

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 7]
```diff
@@ -247,7 +262,21 @@
...
+      "in": "path",
+      "type": "string",
+      "required": true
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "description": "Name of the resource group in which the web service is located.",
+      "in": "path",
+      "type": "string",
+      "required": true
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: Please apply the extension `"x-ms-parameter-location": "method"`, for "ResourceGroupNameParameter" and "WebServiceNameParameter". By default parameters defined in the global parameters section will be properties of the client in the generated code. You would not want resourceGroupName and webServiceName as properties of the client as the customer would like to use the same client for making requests against different resource groups or webservices. Changing the property of the client everytime is inconvenient. Hence please apply the above mentioned extension to these two parameters.

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json [hunk 8]
```diff
@@ -322,33 +351,32 @@
...
         },
         "description": {
           "type": "string",
-          "description": "The description of the Azure ML web service."
+          "description": "The description of the web service."
         },
         "createdOn": {
           "type": "string",
-          "description": "The moment of time the Azure ML web service was created.",
+          "description": "Read Only: The date and time when the web service was created.",
```
Comment (amarzavery) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: @BryanLa - I do not like this aspect of documentation. Why would you want the spec author to duplicate information in the description? The documentation tool should pickup `"readOnly": true` and update the documentation accordingly.

**Review by amarzavery — CHANGES_REQUESTED** [review]

File: arm-machinelearning/2016-05-01-preview/swagger/webservices.json (hunk 8 above)
Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: I mentioned this in the email thread as well, but I agree. I've filed a DocFx tool bug to the VSC team.

Comment (BryanLa) [inline arm-machinelearning/2016-05-01-preview/swagger/webservices.json]: I mentioned this in the email thread as well, but I agree. I've filed a DocFx tool bug to the VSC team.

**Review by amarzavery — APPROVED** [review]

**Pull request merged**